### ISSUE TITLE
Started using timbre ring-logger and seeing a mysterious nil printed at the start of each log line

### ISSUE THREAD

**Issue by JMacLulich** [nberger/ring-logger#24]
Hi Guys,

```
[02/10/17 08:03:05.194] I - nil Finished :get <URL-obscured> for 127.0.0.1 in (38 ms) Status: 200
```

I see the `nil` printed for both `Starting` and `Finished` but I don't see it for the exception output.

Does anyone know where I could look to see about how to figure out what this is supposed to be -- some variable is being printed with value `nil` obviously, but I'm not sure how to find it.

Thanks for your time.

Jason.

**Comment by nberger** [comment]
Hi @JMacLulich 

It seems this is the same issue as in nberger/ring-logger-timbre#6, could you please verify that?
 If this is the case, please note that downgrading to 0.7.5 fixes that issue, that might help until the issue is fixed.

**Comment by JMacLulich** [comment]
Thanks @nberger for that! 

Unfortunately, it seems I am actually running 0.7.5 already:

```
                 [ring-logger-timbre            "0.7.5"]
```

so that's annoying; it seems I will have to suffer the weird output quirk until I have a bit of time to look into how ring-logger is using timbre.

I'll update here if/when I find something.

**Comment by nberger** [comment]
This has been fixed as part of the new release `[ring-logger-timbre "0.7.6"]`. I'll close now but feel free to reopen if you think anything is missing

**Comment by JMacLulich** [comment]
Thanks @nberger for the update.

I actually came back here to update that I'd found where it was happening, but glad that you've released a fix with `0.7.6` I'll update to that.
